### ISSUE TITLE
Given an attribute `a:X` (including `corpus`), set another attribute `b:Y` to every matching item

### ISSUE THREAD

**Issue by benel** [Hypertopic/Porphyry#49]
#### Phase 1
- [ ] Scenarios (Gherkin)
- [ ] Mockups
- [ ] Implementation strategy

#### Phase 2
- [ ] Acceptance tests (Capybara)
- [ ] Implementation

### Phase 3
- [ ] Screencast

**Comment by lerideew** [comment]
For the "Study abroad" project.
It would be interesting for Universities (UTT or others) to be able to change or set item values for different items at a time (e.g. change a degree name on all UE possible in that cursus).

**Comment by benel** [comment]
@lerideew OK... But for the degree, it would be probably a topic, don't you think? Is there an attribute you listed last week that could benefit of such a "search and replace"?

**Comment by lerideew** [comment]
@benel That's true. It can be useful to change a teacher's name if he leaves the University and is replaced by another, the availability, the semester, etc.

**Comment by SorelMboua** [comment]
For the greek vases project, the user should be allowed to set an attribute to all similar items (vases or photography). This feature could be really useful but i don't think it is essential. But that's my opinion, feel free to correct me if i'm wrong.

**Comment by jejroux** [comment]
Projet vitraux:
Compte tenu de la quantité d'items, il peut être utile d'ajouter un attribut à plusieurs items reliés par une même valeur d'attribut en une seule manipulation plutôt que de faire la manipulation pour chaque item concerné.

**Comment by benel** [comment]
@jejroux D'accord... Par exemple, quelle valeur d'attribut à affecter ? Pour les items ayant quel attribut commun ?
Par ailleurs, qui fera cela parmi vos parties prenantes ? Est-on toujours dans la visite sur site ? Ou plutôt à son bureau ou chez soi ?

**Comment by jejroux** [comment]
@benel je ne sais pas trop pour quels attributs cela pourrait être utile... Mais les parties prenantes qui feraient cela seraient les contributeurs de la plateforme,  depuis chez eux.

**Comment by benel** [comment]
> je ne sais pas trop pour quels attributs cela pourrait être utile... Mais les parties prenantes qui feraient cela seraient les contributeurs de la plateforme, depuis chez eux.

Une ressource sur les vitraux ? sur Troyes ? sur ce musée ? N'importe quel exemple qui soit un peu crédible...
